docs/contributors/design.md

```diff
@@ -2,6 +2,13 @@
 
 This is a living document that outlines the design principles and patterns of the editor interface. Its aim is to explain the background of the design, inform future improvements, and help people design great blocks.
 
+
```

Comment:
not sure the extra newlines are needed; they'll probably be strpiped?

Reply:
My lint PR would've caught this one ;-)